eras/conway/impl/src/Cardano/Ledger/Conway/CDDL.hs

```diff
@@ -0,0 +1,971 @@
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+{-# HLINT ignore "Use camelCase" #-}
+{-# HLINT ignore "Evaluate" #-}
+
+module Cardano.Ledger.Conway.CDDL where
+
+import Codec.CBOR.Cuddle.Huddle
+import Data.Function (($))
+import Data.Semigroup ((<>))
+import Data.Text qualified as T
+import Data.Word (Word64)
+import GHC.Float (Double)
+import GHC.Num (Integer)
+import GHC.Show (Show (show))
+
+default (Integer, Double)
+
+conway :: Huddle
+conway = collectFrom [block, transaction]
+
+block :: Rule
```

Comment:
Is the plan for the future to define these in Shelley and to reuse them here (where appropriate)?   And like this to replace our cddl-tool that generates examples?

Reply:
Yes, this should all eventually be modularised - that's the next thing on my TODO list!

Reply:
And we can already generate examples without using the cddl-tool!

Reply:
Aha, I meant we can remove the tool once we have the definitions for all eras. Nice!